### PR TITLE
Force the username to lowercase

### PR DESCRIPTION
**Pull request by MattBlack85**

### CODE REVIEW

File: defender/utils.py [hunk 1]
```diff
@@ -48,14 +48,21 @@ def get_ip(request):
     return ip_address
 
 
+def lower_username(username):
```
Comment (kencochrane) [inline defender/utils.py]: can you add a docstring to this method, so the linters don't complain.

Comment (MattBlack85) [inline defender/utils.py]: sure, adding it

Comment (kencochrane) [inline defender/utils.py]: thank you

**Comment by kencochrane** [comment]
thank you, merging.

**Pull request merged**